imm.yaml

```diff
@@ -0,0 +1,120 @@
+﻿-
+    Name: 'Shared Engineering Standards'
+    Id: 74e29f9b-6dca-4161-8fdd-b468a1eb185d
+    Measures:
+        -
+            Score: 0
```

Comment:
Any reason this and the Coding Standards don't score points?

Reply:
See other comment on initial IMM review as a separate task.